packages/devtools_app/lib/src/shared/console/eval/inspector_tree_v2.dart

```diff
@@ -28,8 +28,16 @@ typedef TreeEventCallback = void Function(InspectorTreeNode node);
 
 const double iconPadding = 4.0;
 const double chartLineStrokeWidth = 1.0;
-double get inspectorColumnWidth => scaleByFontFactor(12.0);
-double get inspectorRowHeight => scaleByFontFactor(16.0);
+double get inspectorColumnIndent => scaleByFontFactor(24.0);
+double get inspectorRowHeight => scaleByFontFactor(20.0);
+
+/// The distance between the expand/collapse and the start of the row, as
+/// determined by a percentage of the [inspectorColumnIndent].
+const expandCollapseToRowStartDistancePercentage = 0.68;
+
+/// The distance between the center of the widget icon and the start of the row,
+/// as determined by a percentage of the [inspectorColumnIndent].
+const iconCenterToRowStartDistancePercentage = 0.15;
```

Comment:
can these go in the class where they are used instead?

Reply:
Yes, done!